src/DynamoCoreWpf/Properties/Resources.en-US.resx

```diff
@@ -2190,7 +2199,7 @@ Uninstall the following packages: {0}?</value>
     <value>Version {0} of {1} could not be found.</value>
   </data>
   <data name="PublishPackageViewPackageHostDependency" xml:space="preserve">
-    <value>External Dependency (optional)</value>
+    <value>Host (optional)</value>
```

Comment:
I would like to revisit this with @hwahlstrom again. I feel this change would eliminate the chance of describing a Dynamo package depending on external software existence rather than integration or host. I think the term `Host` is very limited to the current way Dynamo is integrated, so we did not pick this term in the first place. 

But I guess @hwahlstrom 's starting point is that external dependency may not be trivial enough for users to understand in the first place?

Reply:
I hope we can find the bandwidth to rethink this system - there was a pretty good discussion originally and we discarded some flexible ideas because of time.